python/backend/Dockerfile

```diff
@@ -44,5 +44,10 @@ RUN poetry config virtualenvs.create false \
 # Copy remaining files
 COPY . .
 
+# Check formatting, lint and tests
+RUN poetry run poe format_check \
```

Comment:
can we add a custom poe task `verify` that will run these steps sequentially?

Reply:
sure thing, just updated the PR with it :)